_use-case-gallery/external-recommender/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: External recommender
+subheadline: Example project
+permalink: /use-case-gallery/external-recommender/
+# screenshot: screenshot.png
+# thumbnail: screenshot-thumb.png
+example-project: External+Recommender+Example+Project_2018-12-07_1608.zip
+---
+
+This example provides a ready-to-use project show-casing the use of an *External Recommender* in INCEpTION for annotating named entities
+and part-of-speech tags. Recommenders provide interactive annotation support by suggesting
+potential labels.
+
+External recommenders are web services that expose a machine learning classifier over a HTTP web service.
+They can predict annotations for given documents and optionally be trained on new data. The external recommender
+support in INCEpTION allows to e.g. integrate custom classifiers written in a different programming language
+or to use an already pre-trained model.
+
+The protocol used by INCEpTION to communitcate with an external recommender is documented in the [INCEpTION Developer Guide](https://zoidberg.ukp.informatik.tu-darmstadt.de/jenkins/job/INCEpTION%20(GitHub)%20(master)/de.tudarmstadt.ukp.inception.app$inception-app-webapp/doclinks/3/#_external_recommender_api_overview).
+
+To get started with external recommenders, we provide several external recommender implementations that 
+can serve as examples and as a basis for custom implementations. They are described in the
+following. The links to the GitHub repositories contain documentation installation guides:
+
+## spaCy external recommender
+
+[spaCy](https://spacy.io/) is an open-source software library for advanced Natural Language Processing for 
+Python. It currently offers statistical neural network models for e.g. part-of-speech tagging, dependency
+parsing, named entity recognition in many different languages. The corresponding INCEpTION external recommender
+uses the [Flask](http://flask.pocoo.org/) Python framework to expose POS and NER prediction.
+
+The spaCy recommender is available on [Github](https://github.com/inception-project/external-recommender-spacy).
+
+## Example INCEpTION project
+
+The INCEpTION project for this example contains two documents without annotations. They contain many named 
+entities and tokens with diverse part of speech:
+
+* **Michael\_Jordan\_scientist.txt**: A short article about Michael I. Jordan (scientist).
+* **Wikipedia-Obama.txt**: A short excerpt from the Wikipedia page of Barack Obama.
+
+The following recommenders are configured in this project:
+
+* Named entity value: External recommender
+* Parts-of-speech: External recommender
+
+They are configured with a dummy remote URL and are disabled by default. In order to use them, set up an
+external recommender by either starting an instance of the DKPro TC or spaCy recommender according to the
```

Comment:
Delete mention of DKPro TC here? Instead:

In order to use them, set up an external recommender by e.g. starting an instance of the spaCy recommender ...